pkg/controller/account/ec2.go

```diff
@@ -46,12 +46,23 @@ func (r *ReconcileAccount) InitializeSupportedRegions(reqLogger logr.Logger, acc
 	vCPUQuota, _ := r.getDesiredVCPUValue(reqLogger)
 	reqLogger.Info("retrieved desired vCPU quota value from configMap", "quota.vcpu", vCPUQuota)
 
+	// FIXME: Is this the right place to get it - in the reconcile loop for accounts there are two call to this initialize, so should reduce weird copy-paste errors in the future.
+	// Will however hit the cache for the claim, as non-sts already retrieved the claim before.
+	var kmsKeyId string
+	accountClaim, accountClaimError := r.getAccountClaim(account)
+	if accountClaimError != nil {
+		reqLogger.Error(accountClaimError, "Could not retrieve account claim for account.", "account", account.Name)
```

Comment:
We may not want to be logging an error here? Not all account crs created will have an accountclaim attached at this point, we could just be filling the accountpool, so logging an error may not be necessary